<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ToolForm</class>
 <widget class="QWidget" name="ToolForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>242</width>
    <height>842</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_12">
       <item>
        <widget class="QPushButton" name="loadBtn">
         <property name="text">
          <string>Load Image</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="preProcessCbox">
         <property name="text">
          <string>PreProcess(Media Filter)</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="computerBtn">
       <property name="text">
        <string>Computer</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>Seed Edit</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_11">
          <item>
           <widget class="QRadioButton" name="fRadioButton">
            <property name="text">
             <string>F</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="bRadioButton">
            <property name="text">
             <string>B</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="noneRadioButton">
            <property name="text">
             <string>None</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="clearSeedBtn">
          <property name="text">
           <string>Clear Seed</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>1D Parameters</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Iterations</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="iteration1D">
            <property name="value">
             <number>10</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Foreground Threshold</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="foreThreshold1D">
            <property name="maximum">
             <double>1.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
            <property name="value">
             <double>0.600000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Gamma</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="gamma1D">
            <property name="decimals">
             <number>6</number>
            </property>
            <property name="maximum">
             <double>10.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.000100000000000</double>
            </property>
            <property name="value">
             <double>0.200000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Lambda</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="lambda1D">
            <property name="maximum">
             <double>1000.000000000000000</double>
            </property>
            <property name="value">
             <double>100.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>2D Parameters</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Iterations</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="iteration2D">
            <property name="maximum">
             <number>200</number>
            </property>
            <property name="value">
             <number>10</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>dt</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="dt2D">
            <property name="decimals">
             <number>6</number>
            </property>
            <property name="singleStep">
             <double>0.001000000000000</double>
            </property>
            <property name="value">
             <double>0.010000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Gamma</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="gamma2D">
            <property name="decimals">
             <number>6</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Lambda</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="labmda2D">
            <property name="maximum">
             <double>1000.000000000000000</double>
            </property>
            <property name="value">
             <double>100.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Rendering Contour Threshold</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="renderContourThreshold">
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
         <property name="value">
          <double>0.500000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Segmentation use time</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="computeTime">
         <property name="text">
          <string>10 ms</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>158</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
